### PR TITLE
Update metrics exporter to otel v0.11.0

### PR DESCRIPTION
**Pull request by liiling**
Update metrics exporter and example to otel v0.11.0

### CODE REVIEW

File: exporter/metric/cloudmonitoring.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 	export "go.opentelemetry.io/otel/sdk/export/metric"
 	"golang.org/x/oauth2/google"
 
-	monitoring "cloud.google.com/go/monitoring/apiv3"
+	monitoring "cloud.google.com/go/monitoring/apiv3/v2"
```
Comment (nilebox) [inline exporter/metric/cloudmonitoring.go:29]: This doesn't seem related to updating the otel dependency. Looks like you just updated all dependencies?

We have automation for specifically updating the SDK dependency: please update the version number in https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/ba72e89a40870af14a0bc190e62174ab3e86335a/Makefile#L136
and run `make update-otel`.

**Review by nilebox — CHANGES_REQUESTED** [review]
Thanks for making this change! Could you just update the OTel SDK dependency as mentioned in the comment?

File: exporter/metric/cloudmonitoring.go (hunk 1 above)
Comment (liiling) [reply exporter/metric/cloudmonitoring.go:29]: Thanks for the review! Just updated the SDK dependency as requested. 

For the cloud monitoring API, I updated it because of failure in CircleCI build (it complained that the API is deprecated).

**Review by nilebox — APPROVED** [review]
